### ISSUE TITLE
Create a release of the archived version of the standard

### ISSUE THREAD

**Issue by peterdesmet** [tdwg/tcs#1]
Hi @tdwg/tcs team (including @deepreef @mdoering @baskaufs), I've just migrated _Taxonomic Concept Transfer Schema (TCS)_ from the [TDWG website](http://ww1.tdwg.org/standards/117/) to this repository. The content basically consists of the offered download (now unzipped for better access) and the cover page as the README: https://github.com/tdwg/tcs

I'd now like to archive this version of the standard as a release (= a snapshot of the code, with a version number). Before I can do this, I'd like you to review the README and especially the preferred citation:

> Taxonomic Names and Concepts interest group. 2006. Taxonomic Concept Transfer Schema (TCS), version 1.01. Biodiversity Information Standards (TDWG) http://www.tdwg.org/standards/117

The format of this citation is standardized across TDWG standards, but you might (not required!) want to add individual authors instead of the interest group. The version number can be a date or a number: I chose `1.01` because the download was named `TCS101`, is that OK? I based the year on `date submitted`.

Let me know when everything has been verified. Note: this is about creating an archived version of the standard, any new work (a TCN 2 perhaps?) can happen here in this GitHub repository _after_ I've created the snapshot. That's why we chose GitHub for all TDWG standards: to allow easy, open collaboration.

**Comment by jar398** [comment]
How is this going? Is the citation ready for use in other publications? It looks awfully close.

**Comment by baskaufs** [comment]
The [SDS](http://rs.tdwg.org/sds/doc/specification/) has spelled out a systematic date-based versioning system for all resources. I don't think it is reflected in any of the citations yet.

The issue of authors in citations is still a bit fuzzy. Historically, we have conflated standards and the documents that are included in standards. They are not the same. This is apparent for standards that contain many documents but not as apparent for standards that contain only a single document.  We need to think carefully about whether we actually want to cite the standard or a specific standard document. The "author" will depend on which resource we are talking about. I think that the correct approach is that task groups are the authors of standards, but people are authors of documents (technically contributors). But this hasn't been applied consistently in our documentation yet.

We are working on making IRI dereferencing to human-readable content work for all TDWG resources. All [permanent document IRIs](https://github.com/tdwg/rs.tdwg.org/blob/master/docs/docs.csv) now dereference to a web-accessible human-readable page. So the two TCS documents: <http://rs.tdwg.org/tcs/doc/guide/> and <http://rs.tdwg.org/tcs/doc/xmlschema/> can now be cited by their stable IRIs. 

I'm currently working on the standards landing pages. Eventually, the IRI <http://www.tdwg.org/standards/117> will dereference in a browser to a landing page that will contain links to those two standards documents. At that point, anyone with any standard permanent IRI will be able to find all parts of the standard.

The IRIs in the rs.tdwg.org subdomain also dereference to RDF/XML and RDF/Turtle with appropriate links to other resources. However, there isn't yet a plan for dereferencing the www.tdwg.org IRIs for machine-readable metadata, although those metadata can be retrieved using <http://rs.tdwg.org/dump/standards> using content negotiation or directly using <http://rs.tdwg.org/dump/standards.ttl> or <http://rs.tdwg.org/dump/standards.rdf>.

Basically this is a work in progress, but we have made significant progress in bringing TDWG standards into conformance with the SDS over the past two years.
